controller/src/main/java/io/pravega/controller/server/rest/RESTServer.java

```diff
@@ -68,12 +74,39 @@ protected void startUp() {
         long traceId = LoggerHelpers.traceEnterWithContext(log, this.objectId, "startUp");
         try {
             log.info("Starting REST server listening on port: {}", this.restServerConfig.getPort());
-            httpServer = GrizzlyHttpServerFactory.createHttpServer(baseUri, resourceConfig, true);
+            if (restServerConfig.isEnableTls()) {
+                SSLContextConfigurator sslCon = new SSLContextConfigurator();
```

Comment:
dont we need to specify server auth certificate somewhere for tls.. is the keystorefile the certificate itself?

Reply:
That is correct. keystorefile is the certificate store in `JKS` format.